### PR TITLE
docs(queue): add example to get notification

### PR DESCRIPTION
**Pull request by sileht**
Change-Id: I8ecff82a7e63755b61189bb86f6d23b6e9a84c08

### CODE REVIEW

File: docs/source/actions/queue.rst [hunk 1]
```diff
@@ -519,6 +519,24 @@ enqueued pull requests.
             name: default
 
 
+.._unqueue_notification:
```
Comment (jd) [inline docs/source/actions/queue.rst:522]:
```suggestion
.. _unqueue_notification:
```

File: docs/source/actions/queue.rst [hunk 2]
```diff
@@ -519,6 +519,24 @@ enqueued pull requests.
...
+
+🤙 Get notified when Pull Request is unexpectedly unqueued
```
Comment (jd) [inline docs/source/actions/queue.rst:524]:
```suggestion
🤙 Get notified when a pull request is unexpectedly unqueued
```

**Review by jd — DISMISSED** [review]
see comment

**Comment by jd** [comment]
@mergifyio squash

**Review by jd — APPROVED** [review]

**Review by jd — CHANGES_REQUESTED** [review]
😢

**Comment by sileht** [comment]
> cry

I miss the `Hey @{{author}}` to ping the author :p

**Pull request merged**